admin_manual/release_notes.rst

```diff
@@ -17,6 +18,110 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.9_release_notes_label:
+
+Changes in 10.0.9
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.0.9 that need your attention. You can also read `the full ownCloud Server changelog`_ for further details on what has changed.
+
+New feature: Pending Shares
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ownCloud Server 10.0.9 introduces new features to close usability gaps and to give users more control over incoming shares. Previously shared contents would just appear in the receiving user's file hierarchy and clients would start synchronizing. To provide means for a better management, incoming shares can now have a *pending* state offering the ability to accept or decline (as known from federated sharing). In addition the `recently introduced notifications framework <https://doc.owncloud.com/server/latest/admin_manual/release_notes.html#new-mail-notifications-feature>`_ is being used to inform users via mail. The bell icon in the web interface and the ownCloud Desktop Client can additionally be used to take action. To switch to the new behavior administrators need to disable the configuration option "Automatically accept new incoming local user shares" in the *Sharing* settings section. By default the option will be enabled to preserve the known behavior.
+
+**Please note** that there is no asynchronous batch processing for the mail notifications yet. For this reason ownCloud will send notification mails directly when initiating shares between users. Due to this limitation sharing with large groups (> 50 users) can take some time and might cause load peaks. When operating installations with large groups it is therefore not yet recommended to enable the feature.
```

Comment:
here I suggest to put the limitation part at the beginning of the sentence and the reason "no asynchronous batch processing" at the end of the sentence.

the reason is that someone who just skims through the list will rather see the beginning of sentences but not necessarily the end